clang/lib/Driver/ToolChains/HLSL.cpp

```diff
@@ -283,14 +283,6 @@ HLSLToolChain::TranslateArgs(const DerivedArgList &Args, StringRef BoundArch,
     DAL->append(A);
   }
 
-  // Add default validator version if not set.
```

Comment:
This was removed.  Do we still need to fallback to a default here or does the change in Options.td take care of all of that?

Reply:
The default value in Options.td will take care.